Jenkinsfile

```diff
@@ -11,8 +11,11 @@ pipeline {
         stage('Unit Tests') {
             steps {
                 dir('mhs-reference-implementation') {
-                    sh label: 'Installing dependencies', script: 'pipenv install --deploy --ignore-pipfile'
-                    sh label: 'Running unit tests', script: 'pipenv run unittests'
+                    sh label: 'Installing dependencies', script: 'pipenv install --dev --deploy --ignore-pipfile'
+                    sh label: 'Running unit tests', script: 'pipenv run unittests-cov'
+                    sh label: 'Displaying code coverage report', script: 'pipenv run coverage-report'
+                    sh label: 'Exporting code coverage report', script: 'pipenv run coverage-report-xml'
+                    cobertura coberturaReportFile: '**/coverage.xml'
```

Comment:
For the timebeing, I've added this line to include code coverage reports in Jenkins. Once Sonarqube gets set up, this line could potentially be removed.